pkg/setting/setting.go

```diff
@@ -1372,6 +1372,9 @@ func (cfg *Cfg) readServerSettings(iniFile *ini.File) error {
 	cfg.Domain = valueAsString(server, "domain", "localhost")
 	cfg.HTTPAddr = valueAsString(server, "http_addr", DefaultHTTPAddr)
 	cfg.HTTPPort = valueAsString(server, "http_port", "3000")
+	if os.Getenv("GF_HTTP_PORT") != "" {
```

Comment:
This is already automatically set via `GF_SERVER_HTTP_PORT`, see https://grafana.com/docs/grafana/latest/administration/configuration/#configure-with-environment-variables for reference

Reply:
Cool, thanks!

Reply:
Removed, thanks again - actually I thought to find a better way to set port when experimenting locally with 2 instances behind Nginx - but then forgot about this.